charts/refinery/templates/configmap-config.yaml

```
@@ -6,4 +6,36 @@ metadata:
   {{- include "refinery.labels" . | nindent 4 }}
 data:
   config.yaml: |
-{{- toYaml .Values.config | nindent 4 }}
+    EnvironmentCacheTTL: {{ .Values.config.EnvironmentCacheTTL }}
+    GRPCListenAddr: {{ .Values.config.GRPCListenAddr }}
+    HoneycombAPI: {{ .Values.config.HoneycombAPI }}
+    InMemCollector:
+      CacheCapacity: {{ .Values.config.InMemCollector.CacheCapacity }}
+
+      {{- if hasKey .Values.config.InMemCollector "MaxAlloc" }}
+      MaxAlloc: {{ .Values.config.InMemCollector.MaxAlloc }}
+      {{ else }}
+      MaxAlloc: {{ mulf .Values.resources.requests.memory 0.8 }}
```

We say 75% in README but are multiplying by 0.8 (80%).

```suggestion
      MaxAlloc: {{ mulf .Values.resources.requests.memory 0.75 }}
```

---

Yes, good catch. I actually wanted to ask about this -- I was assuming the more conservative value is best, but happy to change the behavior if needed.